feature-profile/src/main/java/com/codandotv/streamplayerapp/feature_profile/profile/di/ProfilePickerStreamModule.kt

```diff
@@ -1,36 +1,22 @@
 package com.codandotv.streamplayerapp.feature_profile.profile.di
 
 import com.codandotv.streamplayerapp.core_networking.di.QualifierProfileRetrofit
-import com.codandotv.streamplayerapp.feature_profile.profile.data.ProfilePickerStreamRepository
-import com.codandotv.streamplayerapp.feature_profile.profile.data.ProfilePickerStreamRepositoryImpl
 import com.codandotv.streamplayerapp.feature_profile.profile.data.ProfilePickerStreamService
-import com.codandotv.streamplayerapp.feature_profile.profile.domain.ProfilePickerStreamUseCase
-import com.codandotv.streamplayerapp.feature_profile.profile.domain.ProfilePickerStreamUseCaseImpl
-import com.codandotv.streamplayerapp.feature_profile.profile.presentation.screens.ProfilePickerStreamViewModel
-import org.koin.androidx.viewmodel.dsl.viewModel
-import org.koin.dsl.module
+import org.koin.core.annotation.ComponentScan
+import org.koin.core.annotation.Factory
+import org.koin.core.annotation.Module
+import org.koin.core.context.GlobalContext
 import retrofit2.Retrofit
 
-object ProfilePickerStreamModule {
-    val module = module {
-        viewModel {
-            ProfilePickerStreamViewModel(
-                useCase = get()
-            )
-        }
+@Module
+@ComponentScan("com.codandotv.streamplayerapp.feature_profile")
```

Comment:
desconhecimento da minha parte, isso seria tipo id?

Reply:
Isto vai fazer com que todas as annotations (@Single, @Factory etc) que estejam neste package, fiquem vinculadas ("dentro") deste módulo Koin.

Reply:
Mais detalhes em "Como declarar as dependências de cada @Module?" https://dev.to/pedrofsn/conhecendo-o-koin-annotations-5d45